### PR TITLE
Remove non-existed methods from muxrpc manifest

### LINKED ISSUE #272: clock() exposed over muxrpc but isn't exported?
I'm building a tiny SSB CLI and I noticed that this command is throwing an error locally.

### PR DESCRIPTION
**Pull request by christianbundy**
Resolves #272 

We weren't actually exporting this so when experimenting with [ssb-cli](https://github.com/fraction/ssb-cli) there were lots of broken methods showing up.

### CODE REVIEW

**Comment by christianbundy** [comment]
I've also fixed up `help.js`, which was giving incorrect usage info about whether `--seq` was required.

File: help.js [hunk 1]
```diff
@@ -37,12 +33,13 @@ var keysAndValues = {
 
 var MessageId = {
   type: 'string',
-  test: /^%[a-zA-Z0-9\+\/]+={0,2}\.\w+$/
+  test: /^%[a-zA-Z0-9+/]+={0,2}\.\w+$/
```
Comment (christianbundy) [inline help.js]: I removed these escapes because they aren't actually doing anything in this context.

File: help.js [hunk 2]
```diff
@@ -107,7 +105,7 @@ module.exports = {
         },
         seq: {
           type: 'SequenceNumber',
-          optional: false,
+          optional: true,
```
Comment (christianbundy) [inline help.js]: Fixing this unbreaks [`ssb createHistoryStream`](https://github.com/fraction/ssb-cli#usage). :tada:

File: index.js [hunk 3]
```diff
@@ -22,7 +22,6 @@ var manifest = {
   links: 'source',
   add: 'async',
   publish: 'async',
-  getAddress: 'sync',
```
Comment (mixmix) [inline index.js:25]: interestingly, this was not in `v19.1.1`

I've been stepping back through time looking for getAddress references: 
```
$ git checkout HEAD~1 && ack getAddress
```

Comment (mixmix) [reply index.js:25]: adding this back in does not seem to help with the `ssb-server` problem I'm currently having which complains this method doesn't exist

**Pull request merged**